cmd/deployment-check/Makefile

```diff
@@ -1,5 +1,5 @@
 IMAGE="kuberhealthy/deployment-check"
-TAG="v1.7.1"
+TAG="v1.8.2"
```

Comment:
how did this go from 1.7.1 to 1.8.2?? 
when you build and test images, make sure you put them in your own repo / with `-alpha`
We never released 1.8.0 so go ahead and delete the 1.8.0+ images from docker https://hub.docker.com/repository/docker/kuberhealthy/deployment-check

When you build and push like this, you're releasing test images to the public -- be careful please

Reply:
done